hack/charts/mysql-operator/templates/orc-ingress.yaml

```
@@ -5,7 +5,7 @@ kind: Ingress
 metadata:
   name: {{ $fullName }}
   labels:
-    {{ include "mysql-operator.labels" . | indent 4 }}
+{{ include "mysql-operator.labels" . | indent 4 }}
```

Let's use [`nindent`](http://masterminds.github.io/sprig/strings.html#nindent) because it's used all over the code and looks better. 

```suggestion
    {{- include "mysql-operator.labels" . | nindent 4 }}
```